ddp-workspace/projects/ddp-dsm-ui/src/app/medical-record/medical-record.component.ts

```diff
@@ -60,6 +60,7 @@ export class MedicalRecordComponent implements OnInit {
 
   startDate: string;
   endDate: string;
+  hasEndDateChanged = false;
```

Comment:
This has been added so if they manually changed the end date they can over write the default AOM date